css/styles.css

```diff
@@ -1129,15 +1129,14 @@ a.plugin_formcreator_formTile_title {
    clear: both;
    position: fixed;
    bottom: 0;
-   width:  100%;
-   padding: 5px 0;
+   width:  calc(100% - 10px);
 }
 
 .plugin_formcreator_leftHeader#header .plugin_formcreator_leftMenu {
    width: 300px;
    padding-left: 0; /* Overrides CSS from GLPi */
    padding-top: 20px;
-   height: 100%;
+   height: calc(100% - 20px - 65px - 28px); /* - height of top area - padding - footer */
```

Comment:
Same as other comment, depends on other values.

Also this doesn't change anything visually for me (the height of the menu is reduced to not overlap with the footer but it isn't noticeable since the footer was displayed on top of it).

Reply:
I remove it.